src/js/components/Diagram/stories/Progressing.js

```diff
@@ -72,13 +72,20 @@ export const Progressing = () => {
               ))}
             </Box>
           </Box>
-          <Diagram connections={connections} />
+          <Diagram
+            animation={{ type: 'draw', duration: 2000 }}
```

Comment:
This prop is being applied at the component level, however the documentation above is referencing as property part of connections.

Reply:
PR and documentation has been updated to be more accurate on where the Animation prop work is in terms of component level vs connection level.